multiload/main.c

```diff
@@ -73,17 +73,10 @@ help_cb (GtkAction       *action,
 
  	GError *error = NULL;
 
-#if GTK_CHECK_VERSION (3, 22, 0)
```

Comment:
For consistency it would be better to remove only the check blocks that were introduced earlier in this PR.
I'll add the PR which does it for the whole project later.